packages/core/src/managed/overview/PreviewEnvironments.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import { EnvironmentOverview } from './EnvironmentOverview';
+import { EnvironmentsRender, OrderedEnvironments } from '../environmentBaseElements/EnvironmentsRender';
+import { QueryEnvironment } from './types';
+import { getDocsUrl } from '../utils/defaults';
+import { useLogEvent } from '../utils/logging';
+
+interface IPreviewEnvironmentsProps {
+  orderedEnvironments: OrderedEnvironments<QueryEnvironment>;
+  isConfigured?: boolean;
+}
+
+export const PreviewEnvironments = ({ orderedEnvironments, isConfigured }: IPreviewEnvironmentsProps) => {
+  const { environments: previewEnvironments, ...previewEnvironmentsProps } = orderedEnvironments;
+  const logClick = useLogEvent('PreviewEnvironments');
+  const docsLink = getDocsUrl('previewEnvironments');
+  return (
+    <>
+      {(Boolean(previewEnvironments.length) || isConfigured) && (
+        <h4 className="sp-margin-2xl-top sp-margin-m-bottom self-left">
+          <b>Preview Environments</b>
+        </h4>
+      )}
+      {!Boolean(previewEnvironments.length) && isConfigured && (
```

Comment:
Same comment/question here. I find this much harder to read than a simple greater-than comparison.